lib/datapackage/package.rb

```diff
@@ -12,9 +12,9 @@ class Package < Hash
     # package:: Hash or a String
     # schema:: Hash, Symbol or String
     # opts:: Options used to customize reading and parsing
-    def initialize(package = nil, schema = :base, opts = {})
+    def initialize(package = nil, schema = 'data-package', opts = {})
```

Comment:
Just a note that we should remove `schema` argument later because it's now a part of package `descriptor` - `datapackage.profile='data-package'`.